src/Simulator.ml

```diff
@@ -5,7 +5,7 @@ open Utils
 
 let setup (s : SyGuS.t) (z3 : ZProc.t) : unit =
   ignore (ZProc.run_queries ~scoped:false z3 ~db:((
-    ("(set-logic " ^ s.logic ^ ")")
+    (" ")
```

Comment:
This restricts the logic to whatever was specified in the SyGuS file. Without this line, we would ask the SMT solver to auto-decide a logic, which may result in performance degradation. Why do we need that?

Reply:
We do not need that, I reverted to the version where we set logic.